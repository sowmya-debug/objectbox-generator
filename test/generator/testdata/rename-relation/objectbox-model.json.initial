{
  "comment": [
    "KEEP THIS FILE! Check it into a version control system (VCS) like git.",
    "ObjectBox manages crucial IDs for your object model. See docs for details.",
    "If you have VCS merge conflicts, you must resolve them according to ObjectBox docs."
  ],
  "entities": [
    {
      "id": "1:8717895732742165505",
      "name": "TaskRelId",
      "lastPropertyId": "2:6050128673802995827",
      "properties": [
        {
          "id": "1:2259404117704393152",
          "name": "Id"
        },
        {
          "id": "2:6050128673802995827",
          "name": "Group",
          "indexId": "1:501233450539197794"
        }
      ]
    },
    {
      "id": "2:3390393562759376202",
      "name": "TaskRelPtr",
      "lastPropertyId": "2:1774932891286980153",
      "properties": [
        {
          "id": "1:2669985732393126063",
          "name": "Id"
        },
        {
          "id": "2:1774932891286980153",
          "name": "Group",
          "indexId": "2:6044372234677422456"
        }
      ]
    },
    {
      "id": "3:8274930044578894929",
      "name": "TaskRelValue",
      "lastPropertyId": "2:2661732831099943416",
      "properties": [
        {
          "id": "1:1543572285742637646",
          "name": "Id"
        },
        {
          "id": "2:2661732831099943416",
          "name": "Group",
          "indexId": "3:8325060299420976708"
        }
      ]
    },
    {
      "id": "4:7837839688282259259",
      "name": "TaskRelEmbedded",
      "lastPropertyId": "2:5617773211005988520",
      "properties": [
        {
          "id": "1:2518412263346885298",
          "name": "Id"
        },
        {
          "id": "2:5617773211005988520",
          "name": "Group",
          "indexId": "4:2339563716805116249"
        }
      ],
      "relations": [
        {
          "id": "3:6392442863481646880",
          "name": "Groups"
        }
      ]
    },
    {
      "id": "5:7144924247938981575",
      "name": "Group",
      "lastPropertyId": "2:7259475919510918339",
      "properties": [
        {
          "id": "1:161231572858529631",
          "name": "Id"
        },
        {
          "id": "2:7259475919510918339",
          "name": "Name"
        }
      ]
    },
    {
      "id": "6:7373105480197164748",
      "name": "TaskRelManyPtr",
      "lastPropertyId": "1:3287288577352441706",
      "properties": [
        {
          "id": "1:3287288577352441706",
          "name": "Id"
        }
      ],
      "relations": [
        {
          "id": "1:3930927879439176946",
          "name": "Groups"
        }
      ]
    },
    {
      "id": "7:4706154865122290029",
      "name": "TaskRelManyValue",
      "lastPropertyId": "1:2217592893536642650",
      "properties": [
        {
          "id": "1:2217592893536642650",
          "name": "Id"
        }
      ],
      "relations": [
        {
          "id": "2:1929546706668609706",
          "name": "Groups"
        }
      ]
    }
  ],
  "lastEntityId": "7:4706154865122290029",
  "lastIndexId": "4:2339563716805116249",
  "lastRelationId": "3:6392442863481646880",
  "retiredEntityUids": [],
  "retiredIndexUids": [],
  "retiredPropertyUids": [],
  "retiredRelationUids": []
}