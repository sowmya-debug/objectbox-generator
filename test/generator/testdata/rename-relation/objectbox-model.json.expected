{
  "_note1": "KEEP THIS FILE! Check it into a version control system (VCS) like git.",
  "_note2": "ObjectBox manages crucial IDs for your object model. See docs for details.",
  "_note3": "If you have VCS merge conflicts, you must resolve them according to ObjectBox docs.",
  "entities": [
    {
      "id": "1:8717895732742165505",
      "lastPropertyId": "2:6050128673802995827",
      "name": "TaskRelId",
      "properties": [
        {
          "id": "1:2259404117704393152",
          "name": "Id",
          "type": 6,
          "flags": 8193
        },
        {
          "id": "2:6050128673802995827",
          "name": "GroupNew",
          "indexId": "1:501233450539197794",
          "type": 11,
          "flags": 8712,
          "relationTarget": "Group"
        }
      ]
    },
    {
      "id": "2:3390393562759376202",
      "lastPropertyId": "2:1774932891286980153",
      "name": "TaskRelPtr",
      "properties": [
        {
          "id": "1:2669985732393126063",
          "name": "Id",
          "type": 6,
          "flags": 8193
        },
        {
          "id": "2:1774932891286980153",
          "name": "GroupNew",
          "indexId": "2:6044372234677422456",
          "type": 11,
          "flags": 8712,
          "relationTarget": "Group"
        }
      ]
    },
    {
      "id": "3:8274930044578894929",
      "lastPropertyId": "2:2661732831099943416",
      "name": "TaskRelValue",
      "properties": [
        {
          "id": "1:1543572285742637646",
          "name": "Id",
          "type": 6,
          "flags": 8193
        },
        {
          "id": "2:2661732831099943416",
          "name": "GroupNew",
          "indexId": "3:8325060299420976708",
          "type": 11,
          "flags": 8712,
          "relationTarget": "Group"
        }
      ]
    },
    {
      "id": "4:7837839688282259259",
      "lastPropertyId": "2:5617773211005988520",
      "name": "TaskRelEmbedded",
      "properties": [
        {
          "id": "1:2518412263346885298",
          "name": "Id",
          "type": 6,
          "flags": 8193
        },
        {
          "id": "2:5617773211005988520",
          "name": "Group",
          "indexId": "4:2339563716805116249",
          "type": 11,
          "flags": 8712,
          "relationTarget": "Group"
        }
      ],
      "relations": [
        {
          "id": "3:6392442863481646880",
          "name": "GroupsNew",
          "targetId": "5:7144924247938981575"
        }
      ]
    },
    {
      "id": "5:7144924247938981575",
      "lastPropertyId": "2:7259475919510918339",
      "name": "Group",
      "properties": [
        {
          "id": "1:161231572858529631",
          "name": "Id",
          "type": 6,
          "flags": 8193
        },
        {
          "id": "2:7259475919510918339",
          "name": "Name",
          "type": 9
        }
      ]
    },
    {
      "id": "6:7373105480197164748",
      "lastPropertyId": "1:3287288577352441706",
      "name": "TaskRelManyPtr",
      "properties": [
        {
          "id": "1:3287288577352441706",
          "name": "Id",
          "type": 6,
          "flags": 8193
        }
      ],
      "relations": [
        {
          "id": "1:3930927879439176946",
          "name": "GroupsNew",
          "targetId": "5:7144924247938981575"
        }
      ]
    },
    {
      "id": "7:4706154865122290029",
      "lastPropertyId": "1:2217592893536642650",
      "name": "TaskRelManyValue",
      "properties": [
        {
          "id": "1:2217592893536642650",
          "name": "Id",
          "type": 6,
          "flags": 8193
        }
      ],
      "relations": [
        {
          "id": "2:1929546706668609706",
          "name": "GroupsNew",
          "targetId": "5:7144924247938981575"
        }
      ]
    },
    {
      "id": "8:3706853784096366226",
      "lastPropertyId": "2:6745438398739480977",
      "name": "NegTaskRelId",
      "properties": [
        {
          "id": "1:1395437218309923052",
          "name": "Id",
          "type": 0
        },
        {
          "id": "2:6745438398739480977",
          "name": "Group",
          "indexId": "5:2897681629866238117",
          "type": 0
        }
      ]
    },
    {
      "id": "9:2627038740284806767",
      "lastPropertyId": "2:5974317550424871033",
      "name": "NegTaskRelPtr",
      "properties": [
        {
          "id": "1:3398579248012586914",
          "name": "Id",
          "type": 0
        },
        {
          "id": "2:5974317550424871033",
          "name": "Group",
          "indexId": "6:3317123977833389635",
          "type": 0
        }
      ]
    },
    {
      "id": "10:6303220950515014660",
      "lastPropertyId": "2:167566062957544642",
      "name": "NegTaskRelValue",
      "properties": [
        {
          "id": "1:5001958211167890979",
          "name": "Id",
          "type": 0
        },
        {
          "id": "2:167566062957544642",
          "name": "Group",
          "indexId": "7:4778690082005258714",
          "type": 0
        }
      ]
    },
    {
      "id": "11:4035568504096476779",
      "lastPropertyId": "2:6972732843819909978",
      "name": "NegTaskRelEmbedded",
      "properties": [
        {
          "id": "1:1059542851699319360",
          "name": "Id",
          "type": 0
        },
        {
          "id": "2:6972732843819909978",
          "name": "Group",
          "indexId": "8:5558237345453186302",
          "type": 0
        }
      ],
      "relations": [
        {
          "id": "4:7845762441295307478",
          "name": "Groups",
          "targetId": ""
        }
      ]
    },
    {
      "id": "12:959367522974354090",
      "lastPropertyId": "1:771642788862502430",
      "name": "NegTaskRelManyPtr",
      "properties": [
        {
          "id": "1:771642788862502430",
          "name": "Id",
          "type": 0
        }
      ],
      "relations": [
        {
          "id": "5:8514850266767180993",
          "name": "Groups",
          "targetId": ""
        }
      ]
    },
    {
      "id": "13:2914295034816259174",
      "lastPropertyId": "1:8683452355129068124",
      "name": "NegTaskRelManyValue",
      "properties": [
        {
          "id": "1:8683452355129068124",
          "name": "Id",
          "type": 0
        }
      ],
      "relations": [
        {
          "id": "6:4345851588384648695",
          "name": "Groups",
          "targetId": ""
        }
      ]
    }
  ],
  "lastEntityId": "13:2914295034816259174",
  "lastIndexId": "8:5558237345453186302",
  "lastRelationId": "6:4345851588384648695",
  "modelVersion": 5,
  "modelVersionParserMinimum": 5,
  "retiredEntityUids": [],
  "retiredIndexUids": [],
  "retiredPropertyUids": [],
  "retiredRelationUids": [],
  "version": 1
}